snippets/elsif-___.sublime-snippet

```diff
@@ -1,7 +0,0 @@
-<snippet>
-    <content><![CDATA[elsif ${1:condition}
-	$0]]></content>
-    <tabTrigger>elsif</tabTrigger>
```

Comment:
this is valid in crystal too

Reply:
@bew you're right. I removed these since I felt they're not worth having as snippets. The tab trigger is `elsif` and the expanded snippet result in `elsif <cursor here>`.

If we still want these around, I'll keep them. What do you think?

Reply:
indeed, that's pretty useless ^^